src/configs/sys_config.py

```diff
@@ -316,7 +316,7 @@ def get_digit_five_support(num_users: int, domains: List[str] = DIGIT_FIVE):
     "exp_keys": [],
 }
 
-num_users = 9
+num_users = 4
```

Comment:
Please don't make changes to the config other than the ones needed for the feature itself